CONTRIBUTING.md

```diff
@@ -36,7 +34,52 @@ To set up an OpenHTF dev environment, follow the steps below.
 1. `sudo apt-get install python-pip swig libssl-dev python-dev libffi-dev`
 2. `sudo pip install --upgrade pip`
 3. `sudo pip install virtualenv`
-4. `(cd to openhtf directory)`
+4. (cd to openhtf directory)
 5. `virtualenv venv`
 6. `. venv/bin/activate`
 7. `python setup.py develop`
+
+
+## Web Frontend Development
+OpenHTF ships with a built-in web gui found in the `openhtf.io.frontend` module.
```

Comment:
Mention that if they don't want to develop, the pip install action above will include a working frontend.